Doc/library/os.rst

```diff
@@ -3459,6 +3462,82 @@ written in Python, such as a mail server's external command delivery program.
    .. versionadded:: 3.7
 
 
+.. function:: posix_spawnp(path, argv, env, *, file_actions=None, \
+                          setpgroup=None, resetids=False, setsigmask=(), \
+                          setsigdef=(), scheduler=None)
+
+   Wraps the :c:func:`posix_spawnp` C library API for use from Python.
```

Comment:
Please don't duplicate the whole documentation, since it's the same except of the *path* parameter.

I suggest to write a single sentence (I let you reformat it):

```
Similar to :func:`posix_spawn` except that the system searches
for the *executable* file in the list of directories specified by the
``PATH`` environment variable
(in the same way as for ``execvp(3)``)
if the *path* argument contains no directory.
```

But obviously, keep the following markup which is correct:

```
    .. versionadded:: 3.8 
```

Reply:
``` :envvar:`PATH` ```